apps/apps_publisher.py

```diff
@@ -16,9 +16,10 @@
 
 
 class AppsPublisher:
-    def __init__(self, factory, publish_tool: str, registry_host=DockerRegistryClient.DefaultRegistryHost):
+    def __init__(self, factory, publish_tool: str, archs: [], registry_host=DockerRegistryClient.DefaultRegistryHost):
```

Comment:
i think you are mixing types here. `archs` is actually a string or `cmd_exe` would fail. I think this means the test function at the bottom needs to pass `""` instead of `[]`

Reply:
yeah, your're right, originally it was actually a list but then I changed it. Updated via forced push.